examples/existing-resources/version.tf

```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = "1.45.0"
     }
+    restapi = {
+      source  = "Mastercard/restapi"
+      version = ">= 1.17.0"
```

Comment:
examples should lock into lowest version of the range defined in main module. So go with `1.17.0`